package.json

```diff
@@ -11,7 +11,7 @@
     "i18next": "./bin/cli.js"
   },
   "engines": {
-    "node": ">=18.0.0 || >=20.0.0 || >=22.0.0",
+    "node": "^18.0.0 || ^20.0.0 || ^22.0.0",
```

Comment:
This change will match the docs but it will make it impossible to use the lib with node v19 or v21 without passing a flag to ignore engines. If there's undesirable behaviour using `>=18.0.0 <22.0.0 || ^22.0.0` is better.